sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -121,8 +121,10 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                 var azureMonitorOptions = sp.GetRequiredService<IOptionsMonitor<AzureMonitorOptions>>().Get(Options.DefaultName);
                 if (azureMonitorOptions.EnableLiveMetrics)
                 {
-                    var manager = sp.GetRequiredService<LiveMetricsClientManager>();
-                    builder.AddProcessor(new LiveMetricsActivityProcessor(manager));
+                    // TODO: THIS WILL COME FROM THE EXPORTER
+
+                    //var manager = sp.GetRequiredService<LiveMetricsClientManager>();
+                    //builder.AddProcessor(new LiveMetricsActivityProcessor(manager));
```

Comment:
it seems this PR disables live metric support in the distro, but does not enable them in the exporter. If merged, it'd result in a commit in the repo that's know to be broken and can't be released.

Reply:
> this PR disables live metric support in the distro, but does not enable them in the exporter.

This is a chicken-and-egg problem. :)
Exporter code to enable LiveMetrics code hasn't been written yet (next PR) because LiveMetrics isn't available in that project yet (solving with this PR).